libnet/src/route.rs

```diff
@@ -280,7 +287,6 @@ pub fn get_route(destination: IpPrefix) -> Result<Route, Error> {
         typ: sys::RTM_GET as u8,
         version: sys::RTM_VERSION as u8,
         pid: std::process::id() as i32,
-        seq: 47, //TODO
```

Comment:
What was this for?

Reply:
Thats a sequence number for the AF_ROUTE protocol. It's not really used in these simple situations where the socket is opened and closed around a single request and a single reply.